api/src/main/java/jakarta/ejb/ActivationConfigProperty.java

```diff
@@ -35,7 +35,7 @@
  * which messages a JMS message driven bean is to receive.
```

Comment:
"JMS" is a registered trademark, so should also be replaced with the new spec name (or spec short name), so either "Jakarta Messaging" or "messaging"

Reply:
I can certainly do that of course